clang/test/KeyInstructions/init-agg.cpp

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
Use `clang/test/CodeGen/debug-info-*.c`? We use `%clang_cc1` in almost all non-driver tests

Reply:
Moved to `clang/test/DebugInfo/KeyInstructions` as a result of discussion on #134627, but `clang/test/CodeGen/debug-info-*.c` seems sensible too. Happy to move them again, if you have a strong preference?

(updated to use clang_cc1)